.env.example

```
@@ -170,3 +170,7 @@ ENABLE_WEATHER=false
 # Darksky provides an api with 1000 free API calls per day
 # You need to enable the weather above if you provide an API key here.
 DARKSKY_API_KEY=
+
+# Configure rate limits for RouteService per minute
+DEFAULT_RATE_LIMIT_PER_MINUTE_API=60
+DEFAULT_RATE_LIMIT_PER_MINUTE_OAUTH=5
```

better with a newline at the end
```suggestion
DEFAULT_RATE_LIMIT_PER_MINUTE_OAUTH=5

```